api/app/models/spree/webhooks/subscriber.rb

```diff
@@ -26,6 +34,16 @@ def self.with_urls_for(event)
         )
       end
 
+      def self.supported_events(model)
```

Comment:
@aleksandarpetrushev I would rather envision this to be a method that dynamically fetches a list of models (descendants of `Spree::Base` and filters the ones with `HasWebhooks` concern module included) and generates the full list of events for all models